base/docs/basedocs.jl

```diff
@@ -447,6 +447,23 @@ julia> A'
 """
 kw"'"
 
+"""
+    "\$"
```

Comment:
No `" "` here please. This should be just `\$`

Reply:
Why